packages/editor/src/components/page-attributes/check.js

```diff
@@ -20,8 +20,11 @@ export function PageAttributesCheck( { availableTemplates, postType, children }
 }
 
 export default withSelect( ( select ) => {
-	const { getEditedPostAttribute, getEditorSettings } = select( 'core/editor' );
+	const { getEditedPostAttribute } = select( 'core/editor' );
+	const { getEditorSettings } = select( 'core/block-editor' );
 	const { getPostType } = select( 'core' );
+
+	// This setting should not live in the block-editor module.
```

Comment:
`core/editor`?

Reply:
No that's correct, the templates is something post related, and the block editor shouldn't be aware of it (it doesn't need it)